.github/templates/macos_ci_workflow.yml.j2

```diff
@@ -85,11 +86,12 @@ jobs:
           path:
             artifacts.zip
 {%- endif %}
-{% endblock +%}
+{% endblock %}
 {%- if not exclude_test %}
-{% block test +%}
+{% block test %}
```

Comment:
It just came to me that you might've had these issues because you may be using a different jinja version. Could you try pip installing jinja2 for version 3.0.1 and then seeing if these changes are no longer necessary?

Reply:
yep this solved it